src/observable/BoundCallbackObservable.ts

```diff
@@ -138,11 +138,19 @@ function dispatch<T>(state: { source: BoundCallbackObservable<T>, subscriber: Su
   self.add(subject.subscribe(subscriber));
 }
 
-function dispatchNext({ value, subject }) {
+type DispatchNextArg<T> = {
```

Comment:
Usually for signatures used `interface` unexported (https://github.com/ReactiveX/rxjs/blob/dd012796f09e2f76be0d7deb0e7d80310048bcd5/src/operator/windowToggle.ts#L77) - is there difference for this case? I assume interface will works.